src/app/ecosystem-explorer/project-form/services/github/config/repoConfig.ts

```diff
@@ -0,0 +1,5 @@
+export const repoConfig = {
+  owner: 'FilecoinFoundationWeb',
+  repo: 'filecoin-foundation',
+  baseBranch: 'main',
+} as const
```

Comment:
Does this mean we include `baseBranch` everywhere we spread this object?

Reply:
Yes - If that's a problem, we could create two objects ..?

Reply:
It still works with the current setup, so it's not a problem it seems :)